README.md

```diff
@@ -1 +1,168 @@
-# Hello-worl-front-end
\ No newline at end of file
+<h3><b>HELLO WORLD with to apps (front-end)</b></h3>
+  Back-end repo: https://github.com/Ibtisam34/Hello-world-two-rails-react
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [ Hello-world-front-end ] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
```

Comment:
## [OPTIONAL]

- [ ] To make your readme file more professional I think is good that all readme template text descriptions are removed